en/sql/i18n.rst

```diff
@@ -1816,21 +1816,40 @@ Linux
 Timezone library and database compatibility
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The timezone library build by CUBRID includes an MD5 checksum of the timezone data it contains. This hash is stored in all databases which are created with that library, into column **timezone_checksum** of **db_root** system table. If timezone library is changed (recompiled with newer timezone data) and the checksum changes, the CUBRID server, nor any other CUBRID tool cannot be started with existing databases.
+The timezone library built by CUBRID includes an MD5 checksum of the timezone data it contains. This hash is stored in all databases which are created with that library, into column **timezone_checksum** of **db_root** system table. If timezone library is changed (recompiled with newer timezone data) and the checksum changes, the CUBRID server, nor any other CUBRID tool cannot be started with existing databases.
 To avoid such incompatibility, one option is to use /extend argument of the make_tz tool. When this option is provided, the **timezone_checksum** value of database is also changed with the new MD5 checksum of the new timezone library.
+The extend feature should be used when you decide to use a different version of timezone library from the IANA site. It does two things:
+
+-   It generates a new library by merging the old timezone data with the new timezone data. 
+    After the merge, all the timezone regions that aren't present in the new timezone database are kept in order to ensure backward compatibility 
+    with the data in the database tables.
+ 
+-   The second thing that it does is to update the timezone data in the tables in the situation when backward compatibility 
+    could not be ensured in the first phase when the new timezone library was generated. This situation can occur 
+    when an offset rule or a daylight saving rule changes.
+  
+When you run make_tz with the extend option all the databases in your database directory file (databases.txt) are updated together with the MD5 checksum.
+There are some corner cases:
+    
+-   There is the situation when multiple users share the same CUBRID installation and an extend is done by one of them. If that user doesn't have access
+    rights on the files that contain the databases of the other users, those databases will not be updated. After that, if a different user for whom 
+    the update wasn't made will try to do an extend on his databases, he will not be able to do this because the checksum of the library will be 
+    different from then one in his databases.
```

Comment:
I'd like to propose to use gender neutral words. 
he -> "one", "he or she"
his -> "one's", "his or her"

It would be great, if you can also show a solution for the corner cases.